### PR TITLE
DOR-312 Welsh translation

### PR DESCRIPTION
**Pull request by chandaksweta**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -43,18 +52,22 @@ global.error.404.line1=Os gwnaethoch deipio’r cyfeiriad gwe, gwiriwch ei fod y
 global.error.404.line2=Os gwnaethoch ludo’r cyfeiriad gwe, gwiriwch eich bod wedi copïo’r cyfeiriad yn llawn.
 global.error.404.line3=<a class="govuk-link" href="{0}">Anfon eich dogfennau i’w gwirio gan y tollau</a>
 
-global.error.500.title=Sorry, there is a problem with the service
-global.error.500.heading=Sorry, there is a problem with the service
+global.error.500.title=Mae’n ddrwg gennym, mae problem gyda’r gwasanaeth
+global.error.500.heading=Mae’n ddrwg gennym – mae problem gyda’r gwasanaeth
 global.error.500.line1=Try again. If the issue persists and you need help, you can <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="{0}">report the problem (opens in a new window or tab)</a>. We will respond to you within two working days.
```
Comment (Tasmine10ten) [inline conf/messages.cy:57]: The rest of the translations can be found on the story - the newest spreadsheet (the one added yesterday)

File: conf/messages.cy [hunk 2]
```diff
@@ -271,6 +287,26 @@ error.timeOfArrival.hour.invalid-digits=Mae’n rhaid i’r awr cyrraedd gynnwys
...
+error.dateOfDeparture.year.invalid-digits=Mae’n rhaid i’r flwyddyn ymadael gynnwys rhifau yn unig
+error.dateOfDeparture.month.invalid-digits=Mae’n rhaid i’r mis ymadael gynnwys rhifau yn unig
+error.dateOfDeparture.day.invalid-digits=Mae’n rhaid i’r diwrnod ymadael gynnwys rhifau yn unig
+error.dateOfDeparture.all.invalid-value-range=Mae’n rhaid i’r dyddiad ymadael fod rhwng {0} a {1}
+error.dateOfDeparture.all.invalid-value-before-entry-date=Mae’n rhaid i’r dyddiad ymadael fod rhwng {0} (dyddiad mynediad) a {1}
+error.timeOfDeparture.all.required=Nodwch yr amser ymadael amcangyfrifedig
+error.timeOfDeparture.hour.required=Mae’n rhaid i’r amser ymadael gynnwys awr
+error.timeOfDeparture.minutes.required=Mae’n rhaid i’r amser ymadael gynnwys munudau
+error.dateOfArrival.all.invalid-value-range=Mae’n rhaid i’r dyddiad cyrraedd fod rhwng {0} a {1}
+error.dateOfArrival.all.invalid-value-before-entry-date=Mae’n rhaid i’r dyddiad cyrraedd fod rhwng {0} (dyddiad mynediad) a {1}
```
Comment (Tasmine10ten) [inline conf/messages.cy:308]: Two errors missing here, Irene has confirmed we can use: 

error.timeOfDeparture.hour.invalid-digits
=> Mae’n rhaid i’r awr ymadael gynnwys rhifau yn unig

error.timeOfDeparture.minutes.invalid-digits
=> Mae’n rhaid i’r munudau ymadael gynnwys rhifau yn unig

File: conf/messages.cy [hunk 3]
```diff
@@ -343,9 +382,12 @@ form.upload-multiple-files.continue=Yn eich blaen
 error.file-upload.required=Dewiswch ffeil
 error.file-upload.invalid-size-small=Mae’r ffeil dan sylw yn wag
 error.file-upload.invalid-size-large=Mae’n rhaid i’r ffeil dan sylw fod yn llai na {0}MB
-error.file-upload.invalid-type=Mae’n rhaid i’r ffeil dan sylw fod ar ffurf .jpg, .png, .tiff, .pdf, .txt, .msg, Open Document Format (ODF) neu Microsoft Office
+error.file-upload.invalid-type=Mae’n rhaid i’r ffeil dan sylw fod ar ffurf .jpg, .png, .tiff, .pdf, .txt, .msg, Fformat Dogfen Agored (ODF), neu Microsoft Office
 error.file-upload.quarantine=Mae feirws yn y ffeil dan sylw – uwchlwythwch un wahanol
 error.file-upload.unknown=Nid oedd modd uwchlwytho’r ffeil dan sylw
+error.file-upload.remove.try-again=Nid oedd modd tynnu’r ffeil dan sylw – rhowch gynnig arall arni
+error.file-upload.duplicate=Mae’r ffeil dan sylw eisoes wedi’i uwchlwytho
+error.file-upload.try-again=rhowch gynnig arall arni
```
Comment (Tasmine10ten) [inline conf/messages.cy:390]: Still needs to have the previous part of the error so full message will be: 
error.file-upload.try-again= Nid oedd modd uwchlwytho’r ffeil dan sylw  - rhowch gynnig arall arn

Comment (Tasmine10ten) [reply conf/messages.cy:390]: It wouldn't let me comment beside the actual lines but these also need the translation added: 

view.upload-file.help.fileFormat4=Microsoft (Word, Excel or PowerPoint)
view.upload-file.help.fileFormat5=Open Document Format (ODF)

We can use the content from the Beta start page:

**Microsoft (Word, Excel neu PowerPoint)**
**Fformat Dogfen Agored (ODF)**

File: conf/messages.cy [hunk 4]
```diff
@@ -433,7 +481,7 @@ view.beta.paragraph1=Gallwch ddefnyddio’r gwasanaeth hwn i uwchlwytho dogfenna
 view.beta.paragraph2=Mae ond angen i chi anfon dogfennau i CThEM os yw statws eich datganiad yn dangos bod rhaid anfon dogfennau.
 view.beta.paragraph3=Mae llwybrau clirio’r tollau’n cynnwys:
 view.beta.routes.route1=Llwybr 1: gwiriad llawn o’r dogfennau
-view.beta.routes.route1cap=Llwybr 1 CAP (Common Agricultural Policy)
+view.beta.routes.route1cap=Trywydd 1 CAP (Polisi Amaethyddol Cyffredin
```
Comment (Tasmine10ten) [inline conf/messages.cy:484]: Missing a closing bracket. Also I think you were maybe right about using Llwybr (see slack convo). So I think for here **and for the Route page** (still needs updated) we can use: 

Llwybr 1 CAP (Polisi Amaethyddol Cyffredin

File: conf/messages.cy (hunk 1 above)
Comment (Tasmine10ten) [reply conf/messages.cy:57]: Can also get the Hold SLA content from the same spreadsheet - still needs added in

File: conf/messages.cy [hunk 5]
```diff
@@ -433,7 +483,7 @@ view.beta.paragraph1=Gallwch ddefnyddio’r gwasanaeth hwn i uwchlwytho dogfenna
 view.beta.paragraph2=Mae ond angen i chi anfon dogfennau i CThEM os yw statws eich datganiad yn dangos bod rhaid anfon dogfennau.
 view.beta.paragraph3=Mae llwybrau clirio’r tollau’n cynnwys:
 view.beta.routes.route1=Llwybr 1: gwiriad llawn o’r dogfennau
-view.beta.routes.route1cap=Llwybr 1 CAP (Common Agricultural Policy)
+view.beta.routes.route1cap=Trywydd 1 CAP (Polisi Amaethyddol Cyffredin)
```
Comment (Tasmine10ten) [inline conf/messages.cy:486]: Could we also change this one to **Llwybr** 1 CAP (Polisi Amaethyddol Cyffredin)

File: conf/messages.cy [hunk 6]
```diff
@@ -43,18 +52,22 @@ global.error.404.line1=Os gwnaethoch deipio’r cyfeiriad gwe, gwiriwch ei fod y
 global.error.404.line2=Os gwnaethoch ludo’r cyfeiriad gwe, gwiriwch eich bod wedi copïo’r cyfeiriad yn llawn.
 global.error.404.line3=<a class="govuk-link" href="{0}">Anfon eich dogfennau i’w gwirio gan y tollau</a>
 
-global.error.500.title=Sorry, there is a problem with the service
-global.error.500.heading=Sorry, there is a problem with the service
-global.error.500.line1=Try again. If the issue persists and you need help, you can <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="{0}">report the problem (opens in a new window or tab)</a>. We will respond to you within two working days. 
-global.error.500.line2=If you’ve entered any information, we have not saved your answers.
+global.error.500.title=Mae’n ddrwg gennym, mae problem gyda’r gwasanaeth
+global.error.500.heading=Mae’n ddrwg gennym – mae problem gyda’r gwasanaeth
+global.error.500.line1=Rhowch gynnig arall arni. Os yw’r broblem yn parhau a bod angen help arnoch, gallwch [roi gwybod am y broblem (yn agor ffenestr neu dab newydd)]. Byddwn yn eich ateb cyn pen dau ddiwrnod gwaith.
```
Comment (Tasmine10ten) [inline conf/messages.cy:57]: The link itself just shows as text 
image

File: conf/messages.cy (hunk 5 above)
Comment (chandaksweta) [reply conf/messages.cy:486]: Ok, I will change it. Thought Irene asked to leave it as it is..

**Pull request merged**